code/datums/status_effects/debuffs.dm

```diff
@@ -913,6 +913,14 @@
 	examine_text = "<span class='warning'>SUBJECTPRONOUN has a blank, catatonic like stare.</span>"
 	alert_type = /atom/movable/screen/alert/status_effect/ghoul
 
+/datum/status_effect/ghoul/get_examine_text()
+	var/mob/living/carbon/human/H = owner
+	var obscured = H.check_obscured_slots()
+	if(!(obscured & ITEM_SLOT_EYES) && !H.glasses) //The examine text is only displayed if the ghoul's eyes are not obscured
```

Comment:
Checking for glasses here is redundant if you are already checking if eyes are obscured

Reply:
The cult red eyes code also does this. Just checking if obscured still shows the text with glasses on, only not showing it if a mask is worn

Reply:
> Just checking if obscured still shows the text with glasses on

Did you check? does it?

Reply:
> > Just checking if obscured still shows the text with glasses on
> 
> Did you check? does it?

Yes, I tested it. Initially I implemented it with just check_obscured_slots() and it was still showing the text, even with sunglasses, only hiding it when wearing a gas mask.